src/amo/components/AddonMeta.js

```diff
@@ -54,8 +54,10 @@ export class AddonMetaBase extends React.Component {
           <p className="AddonMeta-text AddonMeta-review-count">
             {reviewCount}
           </p>
-          <Rating className="AddonMeta-Rating" rating={averageRating}
-            readOnly styleName="small" />
+          <Rating className="AddonMeta-Rating"
+            rating={averageRating}
+            readOnly
+            styleName="small" />
```

Comment:
Aha looks like there is a rule, I'll add that and use it.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-closing-bracket-location.md